api/src/main/java/org/openmrs/module/openconceptlab/client/OclConcept.java

```diff
@@ -405,6 +405,9 @@ public String toString() {
 		@JsonProperty("allow_decimal")
 		private Boolean allowDecimal;
 
+		@JsonProperty("handler")
+		private String handler;

```

Comment:
I guess we might want to consider calling this "complexHandler"?  But not a strong preference, and I guess it depends on what Burke is calling this.

Reply:
@bmamlin called this "handler", so keeping this as-is